acceptance_tests/features/load_sample.feature

```diff
@@ -3,17 +3,23 @@ Feature: Load sample
    I need to be able to load the sample
    So that a sample file exists in the system
 
+  Background: Internal user is already signed in
```

Comment:
This should be a before_step https://pythonhosted.org/behave/api.html#environment-file-functions